python/plugins/processing/algs/qgis/PointsInPolygonUnique.py

```diff
@@ -81,6 +81,13 @@ def processAlgorithm(self, parameters, context, feedback):
         fieldName = self.getParameterValue(self.FIELD)
         classFieldName = self.getParameterValue(self.CLASSFIELD)
 
+        pointLayerFields=[field.name() for field in pointLayer.fields()]
+        
+        if classFieldName in pointLayerFields:
```

Comment:
You can make this shorter and more explicit with

    if classFieldName not in pointLayerFields: